addons/udes_stock/models/abstract_stock_model.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+_CREATE_OBJECTS = ['group', 'package']
+
+class AbstractStockModel(models.AbstractModel):
```

Comment:
It would be good to have some tests for this class, so they are not needed for each model.

Reply:
Added with products and groups as discussed